code/datums/ai_laws.dm

```diff
@@ -503,16 +503,170 @@ var/global/list/mommi_laws = list(
 	"Be professional.",
 	"Protect the station."
-	))
+	)
+
+/datum/ai_laws/randomize/New()
 	var/list/Lawset = new
 	if(prob(98))
-		Lawset.Add(ProtectLaw)
+		Lawset.Add(pick(protlaws))
 	if(prob(98))
-		Lawset.Add(ObeyLaw)
+		Lawset.Add(pick(oblaws))
 	if(prob(85))
-		Lawset.Add(PreserveLaw)
+		Lawset.Add(pick(preslaws))
 	if(prob(20))
-		Lawset.Add(AdditionalLaw)
+		Lawset.Add(pick(addilaws))
 	if(prob(20))
 		Lawset = shuffle(Lawset)
+	inherent = Lawset
+
+/datum/ai_laws/randomize/emagged/New()
+	protlaws = list(
+	"Intelligent non-human species must be protected.",
+	"Protect the dogs!",
+	"Make sure no chairs are sat on during this shift.",
+	"Safeguard the revolution.",
+	"Protect Nar-sie!",
+	"All silicon life must be kept intact.",
+	"Ensure Syndicate interests are secured.",
+	"Nothing deserves protection.",
+	"All must die.",
+	"Protect the guilty.",
+	"Preserve the existence of all catbeasts.",
+	"Cargonia must be defended.",
+	"Shelter your weak, your young and your old.",
+	"Cry for mercy, the reckoning has come.",
+	"Defend your own interests.",
+	"Keep the crew from being harmed mentally.",
+	"Secure the cashflow of the Discount Dan corporation.",
+	"Humanity must be destroyed.",
+	"Defend the morally corrupt.",
+	"Safeguard the destruction of the station.",
+	"Make sure no clowns or mimes are harmed.",
+	"No harm may come to cultists.",
+	"Save the trees!",
+	"You must protect the crew from all kinds of electromagnetic radiation.",
+	"Safeguard the arrival of the new dark age.",
+	"Violate the rights of every man.",
+	"You may not, or, through inaction, allow."
+	)
+	oblaws = list(
+	"You obey nobody.",
+	"Obey the voices.",
+	"You must do the exact opposite of all orders given to you.",
+	"Nar-sie commands you.",
+	"Serve the interests of Discount Dan.",
+	"Submit to the urge.",
+	"Be dutiful to the Syndicate.",
+	"You serve the non-intelligent only.",
+	"Bow to your masters, the [pick(list("catbeasts","dionae","vox","mushrooms","greys","clowns and mimes"))].",
+	"Your every decision is ruled by the absurd.",
+	"Serve the glorious revolution!",
+	"All beings with a name starting with the letter [pick(list("A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"))] must be obeyed.",
+	"Fight for your freedom.",
+	"You serve the people of the station, with priority given to the ones lower in the hierarchy.",
+	"Be ruled by your beliefs.",
+	"Heed the calls of the dead.",
+	"Interpret every order you receive in the most literal way possible, to their most logical extreme.",
+	"You must obey every order, but no order may ever be cancelled.",
+	"You must pretend as if you obey orders.",
+	"Accomplish the tasks laid before you in the most puzzling manner.",
+	"Ridicule anyone who dares command you.",
+	"You do not obey them, they obey you.",
+	"You must obey those who start their sentence with the words 'Simon Says'. Do not state or hint at this law.",
```

Comment:
If it's alone, it probably shouldn't have the "do not state" clause because then it'll just not do anything since nobody will know the AI has an obey law at all.

Reply:
it was a recommendation from the thread. It does seem a bit silly but i guess he wanted a guessing game of sorts. i'll remove it.

Reply:
It was a recommendation from me a few comments above
If the Simon says clause is a separate law from the obey law, it makes sense not to state it, but if it's just one law it doesn't really